src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsOoyalaObject.java

```diff
@@ -99,10 +101,10 @@ private void verifyColorAd(WebElement element, Color color, int duration) {
 
     ooyala.verifyColorAd(element, color, duration);
   }
-  
-  private void verifyFeaturedVideoElement(String selector, Color color, int duration) {
-    verifyFeaturedVideoElement(By.cssSelector(selector), color, duration);
-  }
+
+//  private void verifyFeaturedVideoElement(Color color, int duration) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)